app/main.go

```diff
@@ -96,9 +105,62 @@ func main() {
 		log.Fatalf("Invalid flags: %v", err)
 	}
 
-	http.HandleFunc("/", index)
-	http.HandleFunc("/sync", sync)
-	http.HandleFunc("/cleanup", cleanup)
+	config := tsbridge.NewConfig(&tsbridge.ConfigOptions{
+		Filename:                 *metricConfig,
+		MinPointAge:              *minPointAge,
+		CounterResetInterval:     *counterResetInterval,
+		SDLookBackInterval:       *sdLookBackInterval,
+		SDInternalMetricsProject: *sdInternalMetricsProject,
+		UpdateParallelism:        *updateParallelism,
+		EnableStatusPage:         *enableStatusPage,
+		StorageEngine:            *storageEngine,
+		SyncPeriod:               *syncPeriod,
+		SyncCleanupAfter:         *syncCleanupAfter,
+	})
+
+	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Instead of wrapping each handler in a function, perhaps you could attach `config` to a struct and make each handler a struct method?

Reply:
Good point, refactored. I've also moved all web ui/tasks stuff into a separate package. PTAL.